src/Uno.UWP/FeatureConfiguration/WinRTFeatureConfiguration.cs

```diff
@@ -19,4 +19,27 @@ public static class ApplicationLanguages
 		/// </summary>
 		public static bool UseLegacyPrimaryLanguageOverride { get; set; } = true;
 	}
+
+	public static class ResourceLoader
+	{
+		// todo@xy: hook up to caching invalidation mechanism
+		/// <summary>
+		/// Use <see cref="ApplicationLanguages.PrimaryLanguageOverride"/> as the override for resource resolution.
+		/// Alternatively if set to false, use <see cref="CultureInfo.CurrentUICulture"/> as the override.
+		/// </summary>
+		public static bool UsePrimaryLanguageOverride { get; set; } = true;
+
+		/// <summary>
+		/// Determines if parsed string resources are preserved between languages change.
+		/// </summary>
+		public static bool PreserveParsedResources { get; set; }
+
+		// todo@xy: not impl
+		// fixme@xy: is this even needed?
+		/// <summary>
+		/// Determines if the ResourceLoader can fallback to the base or one of the sibling culture when a resource fails to resolve with the given languages/cultures.
+		/// Enables resolution between cultures: fr-CA <--> fr-FR, fr <--> fr-CA.
+		/// </summary>
+		public static bool AllowBaseOrSiblingCulturesFallback { get; set; } = true;
```

Comment:
we probably dont need such flag?
legacy uno behave as if this is true, so does windows